### ISSUE TITLE
[FEATURE]: [SHORT DESCRIPTION]

### ISSUE THREAD

**Issue by mikematos84** [jbetancur/react-data-table-component#1159]
## Feature Check list

- [x] Agree to the [Code of Conduct](https://github.com/jbetancur/react-data-table-component/blob/master/CODE-OF-CONDUCT.md)
- [x] Read the README to ensure the feature is not already present
- [x] You read [Creating Issues, Features and Pull Requests](https://github.com/jbetancur/react-data-table-component/issues/387)
- [x] Considered the value versus complexity for all users of the library as well as library maintenance
- [x] Considered if this can be a storybook or documentation example

## Is your feature request related to a problem? Please describe

Allow for disabling of tabbing on empty header cells so they do not get read by screen readers

## Describe the solution you'd like

Allow the passing of `tabindex` to be passed through to `TableCol` cell so the user gains greater flexibility, or simple set `tabindex="-1"` if the content is empty

**Comment by BackupMiles** [comment]
I agree with this issue, I would also include a default behaviour: if a header cell has set "sortable" to false, a `tabindex` should not be applied imho

**Comment by kenirwin** [comment]
The accessibility angle @BackupMiles mentioned is an important one. The "main" issue I'd put in the "it might be nice" column, but for accessibility `tabindex="-1"` when not sortable should be a priority.
